localStorage.js

```diff
@@ -0,0 +1,19 @@
+const fields = document.querySelectorAll('.formData');
+let localData = {
+  fName: '',
+  lName: '',
+  email: '',
+  msg: '',
+};
+window.addEventListener('load', () => {
+  const storedData = localStorage.getItem('portfolio-form-data');
+  fields.forEach((field) => {
+    localData = JSON.parse(storedData);
+    field.value = localData[field.name];
+    field.addEventListener('input', () => {
+      localData[field.name] = field.value;
+      localStorage.setItem('portfolio-form-data', JSON.stringify(localData));
+      console.log(localStorage.getItem('portfolio-form-data'));
+    });
+  });
```

Comment:
Good job so far team, but there are a few changes you need to work on:

- [ ] At the moment, the data is not saved to localStorage as required. Ideally, the form data is stored in localStorage even when the form is not submitted yet. When the form is filled, all the data should be saved to localStorage and then the input field values are again populated with the stored data so that when the page is reloaded, the data from localStorage is on the input fields. All these happen when the form is not submitted. If done correctly, even if the form is submitted, the data from the localStorage is present in the input fields. Kindly work on it. 👍🏽

**See the requirements below**

<img width="924" alt="Screenshot 2022-12-24 at 1 59 01 PM" src="https://user-images.githubusercontent.com/50941074/209439124-28f04d1c-1116-4d1b-8a83-7375f0143fc3.png">